workflows/iuf/operations/preflight-checks-for-services.yaml

```diff
@@ -77,58 +77,71 @@ spec:
                   value: |
                     error_flag=0
 
-                    # check s3 
-                    if ! cray artifacts list boot-images > /dev/null; then
-                      echo "ERROR s3 is not operational."
+                    # check S3
+                    image_list_output=$(cray artifacts list boot-images 2>&1)
+                    if [ $? -ne 0 ]; then
+                      image_list_output=$(echo "$image_list_output" | sed -e 's/^/DEBUG /')
+                      echo "ERROR S3 fails basic functionality test. Please verify it is possible to perform <cray artifacts list boot-images> and try the 'pre-install-check' stage again"
```

Comment:
Let's add to the end of that message something like:

```
See the operations > utility_storage section of the CSM documentation for information on troubleshooting S3 failures.
```

Reply:
Thanks for the doc link @JohnNavitskyHPE . Integrated.